### PR TITLE
Features/civ 5984 check your answers fast track hearing requirements pt2

### PR DESCRIPTION
**Pull request by pascalkankonde**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-5984

### Change description ###

The scope of this ticket is to develop the 'Check your answers before submitting' screen in the scenario where the Defendant has fully admitted and offered to pay by a set date. This screen occurs after the citizen has clicked on the "Notify us about the debt respite scheme" option on the latest update section.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ n] No
```

### CODE REVIEW

File: src/main/common/models/claim.ts [hunk 1]
```diff
@@ -455,6 +456,10 @@ export class Claim {
   get isSmallClaimsTrackDQ(): boolean {
     return this.claimType === claimType.SMALL_CLAIM;
   }
+
```
Comment (laarmada) [inline src/main/common/models/claim.ts:459]: I think we are overload the functionality:
export function analyseClaimType(totalClaimAmount: number): claimType {
  
get isSmallClaimsTrackDQ(): boolean {

 get isFastTrackClaim(): boolean {

at the end 3 function for only one scope to know if is small or fast

File: src/test/unit/services/features/response/checkAnswers/hearingRequirementsSection/buildSmallHearingRequirments.test.ts [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+import {Hearing} from 'models/directionsQuestionnaire/hearing/hearing';
+import {summaryRow} from 'models/summaryList/summaryList';
+import {YesNo} from 'form/models/yesNo';
+import {
+  determinationWithoutHearingQuestion, determinationWithoutHearingReason,
+} from 'services/features/response/checkAnswers/hearingRequirementsSection/buildSmallClaimHearingRequirements';
+import {DeterminationWithoutHearing} from 'models/directionsQuestionnaire/hearing/determinationWithoutHearing';
+
+jest.mock('../../../../../../../main/modules/draft-store');
+jest.mock('../../../../../../../main/modules/draft-store/draftStoreService');
```
Comment (laarmada) [inline src/test/unit/services/features/response/checkAnswers/hearingRequirementsSection/buildSmallHearingRequirments.test.ts:14]: you dont need the draft-store twice...

**Review by laarmada — CHANGES_REQUESTED** [review]

File: src/main/common/models/claim.ts (hunk 1 above)
Comment (TatsyMaslova) [reply src/main/common/models/claim.ts:459]: I don't know Sergio. i prefer it in the claim like that.

File: src/main/services/features/response/checkAnswers/hearingRequirementsSection/buildFastTrackHearingRequirements.ts [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    if(claim.directionQuestionnaire?.hearing?.considerClaimantDocuments.option == YesNo.YES)
+      hearingRequirementsSection.summaryList.rows.push(considerClaimantDocResponse(claim, claimId, lng));
+  }
+  hearingRequirementsSection.summaryList.rows.push(getUseExpertEvidence(claim, claimId, lng));
+  hearingRequirementsSection.summaryList.rows.push(getSentReportToOtherParties(claim, claimId, lng));
+  hearingRequirementsSection.summaryList.rows.push(getShareExpertWithClaimant(claim, claimId, lng));
+  hearingRequirementsSection.summaryList.rows.push(...getExpert(claim, claimId, getLng(lng)));
+  hearingRequirementsSection.summaryList.rows.push(giveEvidenceYourself(claim,claimId,lng));
+
+  console.log(JSON.stringify(hearingRequirementsSection));
```
Comment (TatsyMaslova) [inline src/main/services/features/response/checkAnswers/hearingRequirementsSection/buildFastTrackHearingRequirements.ts:82]: perhaps this needs to be removed

File: src/test/unit/services/features/response/checkAnswers/hearingRequirementsSection/buildFastTrackHearingRequirements.test.ts [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+
+
+describe('Fast Track Claim Hearing Requirements Section', () => {
+  const claimId = 'validClaimId';
+  const lng = 'cimode';
+  const changeButton = 'COMMON.BUTTONS.CHANGE';
+
+  describe('triedToSettleQuestion', () => {
+    it('should return summaryRow if triedToSettle option is no', () => {
+      const claim = new Claim();
```
Comment (TatsyMaslova) [inline src/test/unit/services/features/response/checkAnswers/hearingRequirementsSection/buildFastTrackHearingRequirements.test.ts:39]: Given When Then please

File: src/main/common/models/claim.ts (hunk 1 above)
Comment (laarmada) [reply src/main/common/models/claim.ts]: yes I agree it should be in the claim.ts but the three function have the same scope... I think we can do the same only with only one function...

**Pull request closed without merging**